modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,101 @@
+= Migrating Your PostgreSQL Database with a Kogito upgrade
+:flyway_install_url: https://flywaydb.org/download/community
+:flyway_migrate_existing_url: https://flywaydb.org/documentation/learnmore/existing
+:kogito_ddl_script_url: https://repo1.maven.org/maven2/org/kie/kogito/kogito-ddl
+:flyway_url: https://flywaydb.org/
+:flyway_baseline_migration_url: https://documentation.red-gate.com/fd/baseline-migrations-184127465.html
+
+Kogito provides an out-of-the-box way to manage your database schema changes with each upgrade. It uses link:{flyway_url}[Flyway] to achieve this functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
```

Comment:
```suggestion
When you upgrade your {context} version, by default it will pick up the latest changes available from {context} and apply them automatically.
```